pkg/flow/record.go

```diff
@@ -70,19 +70,6 @@ func NewRecord(
 	}
 }
 
-func Accumulate(r *ebpf.BpfFlowMetrics, src *ebpf.BpfFlowMetrics) {
```

Comment:
So this function `Accumulate` was responsible for merging each core's flows into a single flow, correct?

I'm looking at the same time at this architecture document: https://github.com/netobserv/netobserv-ebpf-agent/blob/main/docs/architecture.md
My understanding is that it still mostly, the overall architecture is the same, there's just a mention of PerCPUHashMap that should be updated, could you do that?

Reply:
done